sklearn/decomposition/tests/test_nmf.py

```diff
@@ -19,15 +20,19 @@
 from sklearn.exceptions import ConvergenceWarning
 
 
-@pytest.mark.parametrize('solver', ['cd', 'mu'])
+@pytest.mark.parametrize(['estimator', 'solver'],
+                         [[NMF, 'cd'], [NMF, 'mu'],
+                          [MiniBatchNMF, 'mu']])
 @pytest.mark.parametrize('regularization',
                          [None, 'both', 'components', 'transformation'])
-def test_convergence_warning(solver, regularization):
+def test_convergence_warning(estimator, solver, regularization):
```

Comment:
nitpick: since `estimator` is a class, I would use `Estimator` (here and there in the tests)

Reply:
For consistency I have changed `estimator` into `Estimator` in all the parametrizations.